types/controllers/index.d.ts

```diff
@@ -62,6 +62,11 @@ export interface IBarControllerDatasetOptions
    */
   barThickness: number | 'flex';
 
+  /**
+   * The base value for the bar in data units along the value axis.
+   */
+  base: number;
```

Comment:
Similar to my comment above, technically all these options are scriptable though it may be that most of these cannot be safely arrays or functions.